plugins/polio/js/src/domains/VaccineModule/SupplyChain/constants.ts

```diff
@@ -17,4 +17,8 @@ export const defaultVaccineOptions = [
         label: 'bOPV',
         value: 'bOPV',
     },
+    {
+        label: 'nOPV2 & bOPV',
+        value: 'nOPV2 & bOPV',
+    },
```

Comment:
This is not correct: supplychain will still handle the vaccines separately. I'll open a separate ticket for that as it will require some backend as well I think